lib/core/src/lib/notifications/components/notification-history.component.ts

```diff
@@ -145,4 +156,8 @@ export class NotificationHistoryComponent implements OnDestroy, OnInit, AfterVie
             this.trigger.closeMenu();
         }
     }
+
+    badgeVisibility(): boolean {
```

Comment:
Did you miss the test case?

Reply:
written 2 test cases for this method. https://github.com/Alfresco/alfresco-ng2-components/pull/9011/files#diff-0d99474bfab4c4b5265ff61b8fb06921dc9f985328a03c7a0b20aedda1283708R188

I will just update the title to include badgeVisibility() method name for clarity

Reply:
updated test titles for badge visibility and added a few tests for intializialition of unreadNotifications https://github.com/Alfresco/alfresco-ng2-components/pull/9011/files#diff-df1e484fec4f3f80a7e230c9ec689d730bb0ad4a5960ee77df151d9fa6923642R64